### PR TITLE
Updated docstrings

### PR DESCRIPTION
**Pull request by consideRatio**
About to adopt the use of hubploy for a deployment for neurohackademy, I had some learning to do about it. I learned some things that I now have embedded into the docstrings.

### CODE REVIEW

File: hubploy/helm.py [hunk 1]
```diff
@@ -1,19 +1,6 @@
...
-    - image/
-    - secrets/
-      - prod.yaml
-      - staging.yaml
-    - config/
-      - common.yaml
-      - staging.yaml
-      - prod.yaml
+Util to deploy a Helm chart (deploy) given hubploy configuration and Helm chart
+configuration located in accordance to hubploy conventions.
```
Comment (consideRatio) [inline hubploy/helm.py:3]: I removed this structure as it is repeated below in the deploy function's docstring and I figure it is one of those that quickly can cause confusion by breaking the DRY principle.

File: hubploy/utils.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
 """
-Utilities for calling out to git
+Utils to extract information (last_modified_commit, path_touched) from the git
+history.
```
Comment (consideRatio) [inline hubploy/utils.py:3]: I don't mention the other functions as they act as helper functions to the two mentioned.

**Comment by consideRatio** [comment]
@salvis2 perhaps you could give this a glance?

File: hubploy/helm.py (hunk 1 above)
Comment (salvis2) [reply hubploy/helm.py:3]: Good call!

**Comment by consideRatio** [comment]
Thanks for the review and merge @salvis2 ! :tada:

**Pull request merged**